templates/bin_deploy

```diff
@@ -8,5 +8,4 @@ branch="$(git symbolic-ref HEAD --short)"
 target="${1:-staging}"
 
 git push "$target" "$branch:master"
-heroku run rails db:migrate --exit-code --remote "$target"
 heroku restart --remote "$target"
```

Comment:
If I am understanding all of this correctly we can get rid of the restarts as well.

Reply:
That's correct!